runelite-client/src/main/java/net/runelite/client/plugins/kourendlibrary/KourendLibraryOverlay.java

```diff
@@ -236,6 +243,20 @@ public Dimension render(Graphics2D g)
 		return null;
 	}
 
+	private boolean isDarkManuscript(@Nullable Book book)
+	{
+		return (book != null && (book == Book.DARK_MANUSCRIPT_13514 || book == Book.DARK_MANUSCRIPT_13515 ||
+				book == Book.DARK_MANUSCRIPT_13516 || book == Book.DARK_MANUSCRIPT_13517 ||
+				book == Book.DARK_MANUSCRIPT_13518 || book == Book.DARK_MANUSCRIPT_13519 ||
+				book == Book.DARK_MANUSCRIPT_13520 || book == Book.DARK_MANUSCRIPT_13521 ||
+				book == Book.DARK_MANUSCRIPT_13522 || book == Book.DARK_MANUSCRIPT_13523));
+	}
+
+	private boolean showVarlamoreEnvoy()
+	{
+		return (Quest.THE_DEPTHS_OF_DESPAIR.getState(client) == QuestState.IN_PROGRESS);
```

Comment:
No need for the surrounding parens.
```suggestion
		return Quest.THE_DEPTHS_OF_DESPAIR.getState(client) == QuestState.IN_PROGRESS;
```

Reply:
Corrected